app/ui.js

```diff
@@ -180,7 +180,6 @@ var UI;
             UI.initSetting('view_only', false);
             UI.initSetting('path', 'websockify');
             UI.initSetting('repeaterID', '');
-            UI.initSetting('token', '');
```

Comment:
We should have a way to add in URL-only parameters.  We should discuss more in a follow-up PR.  For now, this is fine.